Makefile

```diff
@@ -236,3 +236,11 @@ nix/install.sig: nix/install
 		mv $@.tmp $@; \
 	fi
 	touch $@
+
+nixos/options.html: elm
+
+elm: elm-src/index.elm
+	nix-build ./elm-src --out-link ./elm
```

Comment:
Calling `nix-build` here breaks calling the Makefile from a Nix expression (as we do e.g. in https://github.com/NixOS/nixops/blob/master/examples/nix-homepage.nix).

Reply:
But ... what about all the other nix-build calls in the Makefile? 😕